manifests/t/Telegram/TelegramDesktop/4.14.7/Telegram.TelegramDesktop.installer.yaml

```diff
@@ -0,0 +1,39 @@
+# Created with WinGet Releaser main using Komac v1.0.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Telegram.TelegramDesktop
+PackageVersion: 4.14.7
+Platform:
+- Windows.Desktop
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: Telegram\Telegram.exe
+  PortableCommandAlias: telegram
+Scope: user
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Protocols:
+- tg
+ReleaseDate: 2024-01-18
+Installers:
+- Architecture: x86
+  InstallerType: zip
+  InstallerUrl: https://github.com/telegramdesktop/tdesktop/releases/download/v4.14.7/tportable-x64.4.14.7.zip
+  InstallerSha256: 6403914BF3DEEF1644BCF2560EEE82734C18A52AFEEBE5C9F3A8DE0E6ED0D5A5

```

Comment:
```suggestion
```
This installer set has `Architecture: x86` but `tportable-x64` in the `InstallerUrl`.